calculator.js

```diff
@@ -1,28 +1,45 @@
-
 const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+    num1: 3,
+    num2: 5,
+    operation: 'add',
 }
 
-const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+function validNum(num) {
+    if (isNaN(parseInt(num))) {
+        console.log(`${num} is not a number`);
+    }
+    return parseInt(num);
+}
 
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
+function validOperator(operation) {
+    const acceptedOperations = ['add', 'subtract', 'multiply', 'divide', '+', '-', '*', '/']
+    if (!acceptedOperations.includes(operation)) {
+        console.log('Invalid operator');
+    }
+    return operation
 }
```

Comment:
See below for the new way of defining functions

```suggestion
const validOperator = function(operation) {
    const acceptedOperations = ['add', 'subtract', 'multiply', 'divide', '+', '-', '*', '/']
    if (!acceptedOperations.includes(operation)) {
        console.log('Invalid operator');
    }
    return operation
}
```

Reply:
Consider whether it's necessary to return operation from this function. Is it changed?